releasenotes/notes/enable-langgraph-by-default-917c24b82a2db2ef.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    langgraph: The ``langgraph`` integration is enabled by default instead of being opt-in by setting ``_DD_TRACE_LANGGRAPH_ENABLED``.
```

Comment:
Let's clarify that the `_DD_TRACE_LANGGRAPH_ENABLED` env var has been removed.

Reply:
Do we need to add an official release note for the integration itself considering it was previously private feature flag gated?